### PR TITLE
Jd 182 pokemon card hover

### LINKED ISSUE #182: Card hover bug
Fix hover bug in pokemon card where after selling a pokemon, the hover effect malfunctions.

### PR DESCRIPTION
**Pull request by JayDampitan**
## Changes
1. Fix bugs on button animations after selling pokemon cards. 

Closes #182

### CODE REVIEW

**Review by Lap343 — APPROVED** [review]
Nice and easy bug fix

**Review by connielion — APPROVED** [review]
Noice.

File: pokeapp/src/components/PokemonCard.js [hunk 1]
```diff
@@ -38,11 +38,6 @@ const PokemonCard = ({
     return currStyle;
   };
 
-  const hoverMe = () => {
-    setActive(!active);
-    setFirstHover(true);
-  };
-
```
Comment (BlackCubes) [inline pokeapp/src/components/PokemonCard.js:45]: Currently reviewing...........

File: pokeapp/src/components/PokemonCard.js [hunk 2]
```diff
@@ -60,6 +55,15 @@ const PokemonCard = ({
...
   };
 
+  const hoverMe = () => {
+    setActive(true);
+    setFirstHover(true);
+  };
+
+  const leaveHover = () => {
+    setActive(false);
+  };
```
Comment (BlackCubes) [inline pokeapp/src/components/PokemonCard.js:65]: You could do this in one-line to look saucy if you are just returning one thing in the function: `const leaveHover = () => (setActive(false));`

**Review by JorgeLVilla — APPROVED** [review]
Nice job on this Jay!

**Review by BlackCubes — APPROVED** [review]
Lookin' gewd!!

File: pokeapp/src/components/PokemonCard.js (hunk 2 above)
Comment (JayDampitan) [reply pokeapp/src/components/PokemonCard.js:65]: Thank you sir good looking out!

**Pull request merged**